salt/modules/zypper.py

```diff
@@ -628,16 +628,16 @@ def install(name=None,
         log.info('Targeting repo {0!r}'.format(fromrepo))
     else:
         fromrepoopt = ''
+    cmd_install = ['zypper', '--no-refresh', '--non-interactive', 'install',
```

Comment:
I would rather drop the `refresh_db` and add `--no-refresh` only when `refresh` param is set to `False`.